app/assets/stylesheets/_ppt.scss

```diff
@@ -1,6 +1,6 @@
 .button-link {
   background: none !important;
-  color: #2e8aca;
+  color: #1d70b8;
```

Comment:
Why are you having to mess around with defining css?

Reply:
Because the `Save back and come back later` button has to be styled to look like a link :man_shrugging: 
So I went to CDS Exports FE and copied that bit of code. Once I ran Wave it complained that there was a contract issue, so I have just changed to the link colour to the standard GDS one.